lib/chai/utils/proxify.js

```diff
@@ -64,7 +64,7 @@ module.exports = function proxify (obj, nonChainableMethodName) {
         }
       }
 
-      return target[property];
+      return Reflect.get(target, property);
```

Comment:
I don't have much of a preference either way. Anyone have a strong preference here?